src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -291,9 +291,9 @@ trait MarathonConf
   //Internal settings, that are not intended for external use
   lazy val internalStoreBackend = opt[String](
     "internal_store_backend",
-    descr = "The backend storage system to use. One of zk, mesos_zk, mem.",
+    descr = "The backend storage system to use. One of zk, mesos_zk, mem, new_zk, new_mem.",
```

Comment:
I agree. I don't think we can realistically change "zk -> twitter_zk" as it is still the default store. I was tempted to use "legacy_zk". Perhaps the new one should be "zk2"?